spec/models/like_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before(:example) do
+    @user = User.create(name: 'Dev', photo: 'myPhoto', bio: 'myBio', post_counter: 0)
+    @post = Post.create(title: 'myTitle', text: 'Hello', author: @user, comments_counter: 0, likes_counter: 0)
+    @like = Like.new(post: @post)
+    @like.save
+  end
+
+  it 'have correct post' do
+    expect(@like.post_id).to eq(@post.id)
+  end
+
+  it 'test if like counter is updated' do
+    expect(@post.likes_counter).to eq 0
+  end
```

Comment:
- [ ] The entire `Like`  model tests are failing as you can see in the attached image below. Please make the tests green ✔️ 
![image](https://user-images.githubusercontent.com/46839548/229285295-f2e45b07-ca00-4e49-83d6-2dc6fc1e8268.png)
**HINT:** I believe this has something to do with your having `posts_id` instead of `post_id` column in `likes` database table